lib/rom/repository.rb

```diff
@@ -205,12 +213,17 @@ def compile_command(*args, mapper: nil, use: nil, **opts)
 
       if mapper
         mapper_instance = container.mappers[relation.name.relation][mapper]
-      else
+      elsif mapper.nil?
```

Comment:
so, if `mapper == false`  it'll fall through and do nothing.  Desired behavior?

(looks below ... okay, yes, looks to be.)